keeper/registration.go

```diff
@@ -0,0 +1,15 @@
+package keeper
+
+type registration struct {
+	ID                 int32 `gorm:"primary_key"`
+	CheckData          []byte
+	ExecuteGas         uint32 `gorm:"default:null"`
+	LastRunBlockHeight uint64 `gorm:"not null;default:0"`
+	RegistryID         uint32
+	Registry           registry
+	UpkeepID           uint64
+}
+
+func (registration) TableName() string {
+	return "keeper_registrations"
```

Comment:
it was not. "registrations" felt too vague to me. It wouldn't be obvious when looking at the db schema what a registration or a registry was, so the table names have the `keeper_` prefix. But the models are part of the `keeper` package, so it didn't make sense to call it `keeperRegistration` because (I guess this is only an issue if we export them) it would be `keeper.KeeperRegistration`. Such was my line of thinking, happy to change if it feels like an anti-pattern to you.